README.md

```diff
@@ -28,7 +32,7 @@ Then you can use the plugin just like one of TinyMCE's builtin plugins. When con
 `extended_valid_elements: 'span[lang|id] -span'`
 
 # Caveats
```

Comment:
Also, I don't remember if this is called out anywhere in the README, but it might be good to mention that there currently exist no unit tests for this due to difficulty so it has only been tested through manual testing.